### ISSUE TITLE
Contrast issues and docs max-width

### ISSUE THREAD

**Issue by rafaelrenanpacheco** [micronaut-projects/static-website#224]
Hello!

Have you discussed a design option using a lower contrast ratio? In my case, when I open micronaut.io and read the cards, my eyes starts to hurt and I have to stop reading them, and wait a few minutes to try it again.

For instance, Github first official dark theme was the **Default dark**, with high black/white contrast ratio. A few weeks ago they released a **Dark dimmed** version, with lower contrast ratio and way, way better to use and read.

Another suggestion would be to set a max width for the documentation. When reading on a 4k display the documentation gets too wide, which is hard to read since we have to move our head to read. I think there are some studies regarding the best max-width for reading on a display.

What do you think about these two suggestions?

Best regards,
Rafael Pacheco.

**Comment by JasonTypesCodes** [comment]
Thank you for the feedback @rafaelrenanpacheco !  I have shared this with our marketing and design team.
